reports/assets/results.csv

```diff
@@ -1,7 +1,7 @@
 ,dummy,logreg,bayes,svc,random forest
-fit_time,0.978,5.475,1.674,123.556,22.726
-score_time,0.257,0.431,0.369,24.588,0.592
-test_accuracy,0.779,0.655,0.702,0.679,0.784
-test_recall,0.0,0.455,0.342,0.445,0.056
-test_precision,0.0,0.309,0.332,0.331,0.631
-test_f1,0.0,0.368,0.337,0.38,0.102
+fit_time,4.98,33.942,7.986,461.778,255.859
```

Comment:
We should be reproducing the same results on each run of the report, looks like we missed a random state setting somewhere?